nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/EnsureDotNetPackageCorrelation.targets

```diff
@@ -0,0 +1,25 @@
+<Project>
+
+  <PropertyGroup>
+    <PackageCorrelationCliDirectory>$(MSBuildThisFileDirectory)..\DotNetPackageCorrelation.Cli</PackageCorrelationCliDirectory>
+    <DotNetCoreDirectory>$(MSBuildThisFileDirectory)..\..\dotnet-core</DotNetCoreDirectory>
+    <PackageCorrelationFile>$(MSBuildThisFileDirectory)dotnet-package-correlation.json</PackageCorrelationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="$(PackageCorrelationFile)" CopyToOutputDirectory="PreserveNewest" />
+    <_DotNetCoreFiles Include="$(DotNetCoreDirectory)\**\*" />
+  </ItemGroup>
+
+  <Target Name="BuildDotNetPackageCorrelationFile" Inputs="@(_DotNetCoreFiles)" Outputs="$(PackageCorrelationFile)" BeforeTargets="GetCopyToOutputDirectoryItems">
```

Comment:
How does this target get wired up?

Reply:
The `BeforeTargets` attribute inserts it into the build pipeline.

Reply:
Oh I think that was off my screen without me noticing.